tesseract/tesseract_scene_graph/include/tesseract_scene_graph/srdf_model.h

```diff
@@ -0,0 +1,182 @@
+/**
+ * @file srdf_model.h
+ * @brief Parse srdf xml
+ *
+ * @author Levi Armstrong, Ioan Sucan
+ * @date May 12, 2020
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2020, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TESSERACT_SCENE_GRAPH_SRDF_MODEL_H
+#define TESSERACT_SCENE_GRAPH_SRDF_MODEL_H
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <unordered_map>
+#include <string>
+#include <vector>
+#include <utility>
+#include <memory>
+#include <tinyxml2.h>
+#include <console_bridge/console.h>
+#include <tesseract_scene_graph/graph.h>
```

Comment:
graph.h shouldn't be inside the push/pop

Reply:
Fixed